examples/pycaffe/layers/pascal_multilabel_datalayers.py

```diff
@@ -0,0 +1,262 @@
+# imports
+import json, time, pickle, scipy.misc, skimage.io, caffe
+
+import numpy as np
+import os.path as osp
+
+from xml.dom import minidom
+from random import shuffle
+from threading import Thread
+from PIL import Image
+
+from pascal_multilabel_with_datalayer_tools import SimpleTransformer
```

Comment:
Ok, sure, that makes sense. That's a good compromize. :). Do you want to make a new pull requests, or should I edit yours?